master/pkg/stream/stream_test.go

```diff
@@ -0,0 +1,121 @@
+package stream
+
+import (
+	"testing"
+
+	"github.com/gorilla/websocket"
+	"github.com/stretchr/testify/require"
+)
+
+type BroadcastVerification struct {
+	Upserted []*websocket.PreparedMessage
+	Deleted  []*websocket.PreparedMessage
+}
+
+type TestMsg struct {
```

Comment:
Great move on using a TestMsg! This makes sure these test are focused on a type-independent approach rather than being tied into Trials